fixtures/parallelizer/remote.py

```diff
@@ -213,8 +213,8 @@ def _init_config(slave_options, slave_args):
     appliance_data = conf.slave_config.get("appliance_data", {})
     if ip_address in appliance_data:
         template_name, provider_name = appliance_data[ip_address]
-        conf.runtime["cfme_data"]["basic_info"]["appliance_template"] = template_name
-        conf.runtime["cfme_data"]["basic_info"]["appliances_provider"] = provider_name
+        conf.cfme_data["basic_info"]["appliance_template"] = template_name
+        conf.cfme_data["basic_info"]["appliances_provider"] = provider_name
```

Comment:
That should not be happening, and is a conf bug. This might actually be fixed in the yaycl branch, so I'll get that sorted and and also get a test covering this case (which yaycl *might* already have).